frontend/demo/component/menubar/menu-bar-custom-theme.ts

```diff
@@ -1,3 +1,36 @@
-import '../../init';
+import 'Frontend/demo/init'; // hidden-source-line
 
-import '@vaadin/vaadin-menu-bar/vaadin-menu-bar';
+import { html, LitElement } from 'lit';
+import { customElement, state } from 'lit/decorators.js';
+import '@vaadin/menu-bar';
+import { applyTheme } from 'Frontend/generated/theme';
+
+@customElement('menu-bar-custom-theme')
+export class Example extends LitElement {
+  protected createRenderRoot() {
+    const root = super.createRenderRoot();
+    // Apply custom theme (only supported if your app uses one)
+    applyTheme(root);
+    return root;
+  }
+
+  // tag::snippet[]
+  @state()
+  private items = [
+    { text: 'View', theme: 'custom-theme' },
+    { text: 'Edit' },
+    {
+      text: 'Share',
+      children: [{ text: 'By email', theme: 'custom-theme' }, { text: 'Get link' }],
+    },
+  ];
+  // end::snippet[]
+
+  render() {
+    return html`
+      <!-- tag::snippethtml[] -->
+      <vaadin-menu-bar .items="${this.items}"></vaadin-menu-bar>
+      <!-- end::snippethtml[] -->
+    `;
+  }
```

Comment:
I would extend the snippet to cover the `render()` method as well, makes it clearer IMO.

```suggestion

  render() {
    return html`
      <vaadin-menu-bar .items="${this.items}"></vaadin-menu-bar>
    `;
  }
  // end::snippet[]
```

Reply:
That is aligned with other examples:
https://github.com/vaadin/docs/blob/0655cdd8a8316e9620516b0ac068f0d1b253cbe0/frontend/demo/component/menubar/menu-bar-basic.ts#L47
Would it make sense to update it for other examples as well? 🤔

Reply:
Yes, please, update the other examples as well. But maybe in another PR?